ledger/sandbox-common/src/test/lib/scala/platform/sandbox/services/reset/ResetServiceITBase.scala

```diff
@@ -184,7 +208,7 @@ abstract class ResetServiceITBase
           .map(ids => ids.distinct should have size 20L)
       }
 
-      // 5 attempts with 5 transactions each seem to strike the right balance to complete before the
+      // 4 attempts with 5 transactions each seem to strike the right balance to complete before the
```

Comment:
Why this change?

Reply:
Sorry, forgot to copy this from the commit body.

Now we're also checking for ledger configuration, the time for a reset to be "accepted" is longer, so 5 tries in 30 seconds is a bit flaky. Reducing to 4 seems reasonable.